std/format.d

```diff
@@ -89,7 +89,7 @@ private alias enforceFmt = enforceEx!FormatException;
    Interprets variadic argument list $(D args), formats them according
    to $(D fmt), and sends the resulting characters to $(D w). The
    encoding of the output is the same as $(D Char). The type $(D Writer)
-   must satisfy $(D $(REF isOutputRange, std,range,primitives)!(Writer, Char)).
+   must satisfy $(REF isOutputRange, std,range,primitives)$(D !(Writer, Char)).
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4370550/23589629/c0814928-01d0-11e7-9893-17a816f14145.png)

However, it always looked like this: http://dlang.org/phobos/std_format.html